labelbench/storage/database.py

```
@@ -96,11 +114,19 @@ def _init_db(self):
+                    status TEXT DEFAULT 'draft',
                     annotated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
                     FOREIGN KEY (sample_id) REFERENCES samples(id) ON DELETE CASCADE
                 )
             """)
             
+            # Add status column if it doesn't exist (migration for existing databases)
+            try:
+                conn.execute("ALTER TABLE annotations ADD COLUMN status TEXT DEFAULT 'draft'")
+            except Exception:
+                # Column already exists, ignore
+                pass
```

Ruff (0.14.7)

`try`-`except`-`pass` detected, consider logging the exception

(S110)